commands/login/index.js

```diff
@@ -1,28 +1,28 @@
 const error = require('error/typed');
 const p = require('commander');
-const program = new p.Command('losant configure');
+const program = new p.Command('losant login'); // TODO why was that like that?
```

Comment:
probably want to resolve this todo?

Reply:
pretty sure this is just an old bad copy paste from past Erin.